builtin/logical/ssh/fields.go

```diff
@@ -0,0 +1,41 @@
+package ssh
+
+import "github.com/openbao/openbao/sdk/v2/framework"
+
+func addSubmitIssuerCommonFields(fields map[string]*framework.FieldSchema) map[string]*framework.FieldSchema {
```

Comment:
I do rather like this pattern, since it keeps `path_*.go` files shorter. Thanks, @Gabrielopesantos!